packages/components/src/Link/Link.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { Link } from ".";
+
+afterEach(cleanup);
```

Comment:
This isn't needed - Jest does this automatically. (See https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup)